### PR TITLE
[EN] Fixed formula in English documents for chapter 13-3, Notation section.

### PR DESCRIPTION
**Pull request by xcastilla**
The formula wasn't correctly rendered:
Before fix:
Screen Shot 2020-09-19 at 11 16 45
After fix:
Screen Shot 2020-09-19 at 11 17 28

I haven't checked the rest of the translations, but since the English one wasn't fixed maybe the rest do have this problem as well.

@Atcold

### CODE REVIEW

**Review by Atcold — CHANGES_REQUESTED** [review]
`j` should be left outside the bold.
Also, add "Close #563" to your commit message, so that you can close the issue automatically.

File: docs/en/week13/13-3.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Unlike a sequence, it does not have an order.
 </center>
 
 In Figure 1, vertex $v$ is comprised of two vectors: input $\boldsymbol{x}$ and its hidden representation $\boldsymbol{h}$.
-We also have multiple vertices $v_{j}$, which is comprised of $\boldsymbol{x}_{j}$ and $\boldsymbol{h}_{j}$.
+We also have multiple vertices $v_{j}$, which is comprised of $\boldsymbol{x_{j}}$ and $\boldsymbol{h_{j}}$.
```
Comment (Atcold) [inline docs/en/week13/13-3.md:34]:
```suggestion
We also have multiple vertices $v_{j}$, which is comprised of $\boldsymbol{x}\_j$ and $\boldsymbol{h}\_j$.
```

**Review by Atcold — CHANGES_REQUESTED** [review]
Here, try this.

**Comment by Atcold** [comment]
You can tell the `_` is triggering the italic.

image

**Comment by xcastilla** [comment]
My bad, I didn't see the open issue. I commited the requested change.

**Review by Atcold — COMMENTED** [review]
Did you try running the website locally? Did it fix the issue?

**Comment by xcastilla** [comment]
This is what the rendered version looks like after applying your suggestion. The subscript has been left outside the bold correctly.
Screen Shot 2020-09-19 at 11 50 09

**Review by Atcold — APPROVED** [review]
Great! Thanks!

**Comment by Atcold** [comment]
@xcastilla, can we fix this for all the other languages as well? Thanks!

**Pull request merged**